shared/src/main/java/com/couchbase/lite/WhereRouter.java

```diff
@@ -14,6 +14,6 @@
 
 package com.couchbase.lite;
 
-/* package */  interface WhereRouter {
-    Where where(Expression expression);
+interface WhereRouter {
+    Where where(Expression expr);
```

Comment:
I think it's better to make the variable name inline with the spec. Otherwise, each platform will have something totally different (I means API Doc / Guide). If you prefer short version, please create an issue to the spec and we will fix the spec and code at the same time. 

FYI, I have to go back and fix iOS code several times when I found that I don't have the name according to the spec.